### PR TITLE
Fix PyPy support.

### LINKED ISSUE #4: client_for is incompatible with pypy
```
tests/tornado/test_hyperbahn.py:62:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tchannel/tornado/tchannel.py:116: in __init__
    self.hooks.register(ZipkinTraceHook(tchannel=self))
tchannel/zipkin/zipkin_trace.py:44: in __init__
    self.tracer = TChannelZipkinTracer(tchannel)
tchannel/zipkin/tracers.py:141: in __init__
    TCollectorClient = client_for('tcollector', TCollector)
tchannel/thrift/client.py:96: in client_for
    service_module, thrift_service_name, method_name
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

service_module = <module 'tchannel.zipkin.thrift.TCollector' from '/Users/blampe/src/tchannel/python/tchannel/zipkin/thrift/TCollector.pyc'>
service_name = 'TCollector', method_name = '__delattr__'

    def generate_method(service_module, service_name, method_name):
        """Generate a method for the given Thrift service.

        :param service_module:
            Thrift-generated service module
        :param service_name:
            Name of the Thrift service
        :param method_name:
            Method being called
        """
        assert service_module
        assert service_name
        assert method_name

>       args_type = getattr(service_module, method_name + '_args')
E       AttributeError: 'module' object has no attribute '__delattr___args'

tchannel/thrift/client.py:116: AttributeError
```

### PR DESCRIPTION
**Pull request by abhinav**
Resolves #4.

### CODE REVIEW

**Comment by abhinav** [comment]
CC @blampe

File: tchannel/thrift/util.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    :param iface:
+        The Thrift-generated Iface class defining the interface for the
+        service.
+    :returns:
+        A set containing names of the methods defined for the service.
+    """
+    methods = inspect.getmembers(iface, predicate=inspect.ismethod)
+
+    return set([
```
Comment (blampe) [inline tchannel/thrift/util.py]: No need for the list comp. `set(name for ...)`.

**Comment by blampe** [comment]
:+1:

File: tchannel/thrift/util.py (hunk 1 above)
Comment (abhinav) [inline tchannel/thrift/util.py]: Fixed

**Pull request merged**